### PR TITLE
[#683] Convert commands/packageManager.ts to TS

### PR DESCRIPTION
**Pull request by moriyuu**
Summary:
---------

Part of #683
Converted `commands/packageManager.ts` to typescript

Test Plan:
----------

`yarn test`

### CODE REVIEW

File: packages/cli/src/tools/packageManager.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
 // @flow
```
Comment (thymikee) [inline packages/cli/src/tools/packageManager.ts]: Please remove Flow annotation

File: packages/cli/src/tools/packageManager.ts [hunk 2]
```diff
@@ -29,8 +27,8 @@ const packageManagers = {
 
 function configurePackageManager(
   packageNames: Array<string>,
-  options?: Options,
   action: 'install' | 'installDev' | 'installAll' | 'uninstall',
+  options?: Options,
```
Comment (thymikee) [inline packages/cli/src/tools/packageManager.ts]: @Esemesek you ok with that? Makes sense imho

**Comment by thymikee** [comment]
Please make sure lint passes

**Comment by moriyuu** [comment]
@thymikee Should I convert `tools/__tests__/packageManager-test.js` too?

**Comment by thymikee** [comment]
yes please!

File: packages/cli/src/tools/packageManager.ts (hunk 2 above)
Comment (Esemesek) [reply packages/cli/src/tools/packageManager.ts]: Yup. It makes a lot of sense.

**Pull request merged**